system/jlib/jutil.cpp

```diff
@@ -2505,6 +2528,23 @@ jlib_decl bool queryDafsSecSettings(DAFSConnectCfg *_connectMethod,
     return ret;
 }
 
+//query PKI values from environment.conf
+jlib_decl bool queryHPCCPKIKeyFiles(const char * *  _certificate,//HPCCCertFile
+                                    const char * *  _privateKey, //HPCCPrivateKeyFile
+                                    const char * *  _passPhrase) //HPCCPassPhrase
+{
+    bool rc = true;
+    if (HPCCcertificate.isEmpty() && HPCCprivateKey.isEmpty() && HPCCpassPhrase.isEmpty())
+        rc = querySecuritySettings(nullptr, nullptr, nullptr, nullptr, nullptr);
```

Comment:
querySecuritySettings() checks for dfsUseSSL which does not seem to matter for a general purpose routine.
Can we have a more general routine that both the old querySecuritySettings() and the new routine could call to get the Certificate/PrivateKeyFile/PassPhrase values ?